site2/website-next/docusaurus.config.js

```diff
@@ -143,54 +142,6 @@ module.exports = {
           position: "right",
           label: "Docs",
         },
-        {
-          label: "REST APIs",
-          position: "right",
-          items: [
-            {
-              label: "Admin REST API",
-              to: "/admin-rest-api",
-            },
-            {
-              label: "Functions",
-              to: "/functions-rest-api",
-            },
-            {
-              label: "Sources",
-              to: "/source-rest-api",
-            },
-            {
-              label: "Sinks",
-              to: "/sink-rest-api",
-            },
-            {
-              label: "Packages",
-              to: "/packages-rest-api",
-            },
-          ],
-        },
-        {
-          label: "CLI",
-          position: "right",
-          items: [
-            {
-              label: "Pulsar Admin",
-              to: "/pulsar-admin-cli",
-            },
-            {
-              label: "Pulsar Client",
-              to: "/pulsar-client-cli",
-            },
-            {
-              label: "Pulsar Perf",
-              to: "/pulsar-perf-cli",
-            },
-            {
-              label: "Pulsar",
-              to: "/pulsar-cli",
-            },
-          ],
-        },
```

Comment:
why remove this? it's needed.

Reply:
@urfreespace  On the website weekly meeting yesterday, it was been decided that these links will be added to the docs and removed from the nav.